src/main/java/fact/TriggerEmulation/Discriminator.java

```diff
@@ -0,0 +1,120 @@
+package fact.TriggerEmulation;
+
+import fact.Constants;
+
+/**
+ *Class with functions to discriminate the signals of a given patch (or a bunch of them).
+ * Furhtermore it contains helper functions for the discrimination e.g. conversion between DAC and mV
+ */
+public class Discriminator {
+
+    public static int default_slice = 0;
+
+    /**
+     *Discriminate the signal of a given patch
+     * @param data timeseries, array[n_pixels_pe_patch]
+     * @param threshold threshold of the discriminator in DAC
+     * @param minTimeOverThreshold minimum time the signal has to stay above the threhold
+     * @param skipFirst number of slices to ignore at the beginning of the time series
+     * @param skipLast number of slices to ignore at the end of the time series
+     */
+    public static int discriminatePatch(
+            double[] data,
+            int threshold,
```

Comment:
you convert `threshold` from dac to mV further down, so this `threshold` could be named `threshold_in_dac` to make the unit clear for an outside user of this function.